app/controllers/workorders_controller.rb

```diff
@@ -8,6 +21,10 @@ def new
 
   def create
     @workorder = Workorder.new workorder_params
+    if params[:id]
+      @workorder.update(help_request_id: params[:id])
+      HelpRequest.find(params[:id]).update(workorder_id: @workorder.id)
```

Comment:
This feels fishy. Isn't this `id` related to the workorder? How can it be used to find a `HelpRequest`? Make sure the fact that both ids are 1 in the test is not causing problems here.

If id does relate to the `HelpRequest`, we should probably go ahead and call it `help_request_id`